engine/server/engine/main.go

```diff
@@ -121,19 +120,6 @@ func runMain() error {
 
 	logsDatabaseClient := kurtosis_backend.NewKurtosisBackendLogsDatabaseClient(kurtosisBackend)
 
-	engineServerService := server.NewEngineServerService(serverArgs.ImageVersionTag, enclaveManager, serverArgs.MetricsUserID, serverArgs.DidUserAcceptSendingMetrics, logsDatabaseClient)
```

Comment:
If we're not instantiating that guy anymore, can we remove the associated file? Or it's used somewhere else?

Reply:
yepp; will do that.